embark-ui/src/index.css

```diff
@@ -88,4 +88,14 @@
 
 .pointer {
   cursor: pointer;
+}
+
+table tr.error {
+  background-color: rgba(255, 0, 0, 0.09);
+  color: rgba(255, 0, 0, 0.75);
+  line-height: 2.2rem;
+}
```

Comment:
I think we can use one of CoreUI's existing alerts https://coreui.io/docs/components/alerts/

Reply:
Updated.